modules/5_install/templates/keepalived_vrrp_instance.tpl

```diff
@@ -0,0 +1,16 @@
+vrrp_instance OCP_PLB {
+  state <STATE>
+  interface <INTERFACE>
+  virtual_router_id ${virtual_router_id}
+  priority 100
```

Comment:
@yussufsh Seems the same priority will be set on all the nodes, as per keepalived guidelines we should have lower priority on Backup nodes compared to Master.

Reply:
Good point @Prajyot-Parab  I will push the changes required for public vrrp.